data/openshift.yml

```diff
@@ -95,6 +95,9 @@ openshift:
     Create cluster - Run on bare metal - User provisioned infrastructure:
       url_rules:
         - /openshift/install/metal/installer-provisioned
+    Create cluster - Create an OpenShift Cluster - Nutanix AOS:
```

Comment:
Let's add also tracking for the direct page to Nutanix with Installer-provisioned.
This may give us more information if users bookmark this link .


```
    Create cluster - Create an OpenShift Cluster - Nutanix AOS - Installer provisioned infrastructure:
    url_rules:
      - /openshift/install/nutanix/installer-provisioned
```

Please mind the YAML linting, it may not be correct in the code snippet.

Reply:
Fixed in https://github.com/RedHatInsights/pendo-config-automation/pull/186/commits/7b0ab77978250c25d890a9ea46be8ea58c683ae4